stubs/seaborn/seaborn/axisgrid.pyi

```diff
@@ -60,7 +211,7 @@ class FacetGrid(Grid):
     hue_kws: dict[str, Any]
     def __init__(
         self,
-        data: DataFrame,
+        data: DataFrame | _SupportsDataFrame,
```

Comment:
seaborn version 0.13 started supporting other data frames that support the `__dataframe__` protocol: https://seaborn.pydata.org/whatsnew/v0.13.0.html#support-for-alternate-dataframe-libraries